src/pinetree/polymer.cpp

```diff
@@ -112,10 +112,12 @@ Polymer::Polymer(const std::string &name, int start, int stop)
 }
 
 Polymer::~Polymer() {
-  // Error check to make sure nothing is uncovered
-  // and no polymerases are bound
+  // The following code can be uncommented to check that
+  // polymers are being properly destroyed during the simulation,
+  // i.e. nothing is left uncovered/ no polymerases are bound. 
+  // For debugging purposes only. 
   //
-  // std::cout << "Begin destroying polymer." << std::endl;
+  /*std::cout << "Begin destroying polymer." << std::endl;
```

Comment:
I commented this section out because it now prints a bunch of warnings after the simulation terminates, since now all of the model stuff is being properly destroyed. These warnings don't matter after the simulation runs successfully, and also the end user probably shouldn't ever see them anyway (it's really more of a debugging tool for developers).

I think it would be better to encompass this error checking in a unit test/integration test if possible, but I haven't written one yet so I'm just leaving it commented out with a note for now.